src/main/java/magicbees/main/utils/compat/bloodmagic/ItemFrenziedFrame.java

```diff
@@ -0,0 +1,138 @@
+            return 10;
+        }
+
+        @Override
+        public float getLifespanModifier(IBeeGenome genome, IBeeGenome mate, float currentModifier) {
+            return 0.0001f;
+        }
+
+        @Override
+        public float getProductionModifier(IBeeGenome genome, float currentModifier) {
+            return 0;
+        }
+
+        @Override
+        public float getFloweringModifier(IBeeGenome genome, float currentModifier) {
+            return 0;
+        }
+
+        @Override
+        public float getGeneticDecay(IBeeGenome genome, float currentModifier) {
+            return 10;
+        }
+
+        @Override
+        public boolean isSealed() {
+            return false;
+        }
+
+        @Override
+        public boolean isSelfLighted() {
+            return true;
+        }
+
+        @Override
+        public boolean isSunlightSimulated() {
+            return false;
+        }
+
+        @Override
+        public boolean isHellish() {
+            return false;
+        }
+
+        @Override
+        public float getTerritoryModifier(IBeeGenome genome, float currentModifier) {
+            return 1;
+        }
+    }
+
+    @Override
+    public void addInformation(ItemStack par1ItemStack, EntityPlayer par2EntityPlayer, List par3List, boolean par4) {
+        par3List.add(StatCollector.translateToLocal("MAY CHAOS TAKE THE WORLD"));
+
+        if (!(par1ItemStack.getTagCompound() == null)) {
+            par3List.add(
+                    StatCollector.translateToLocal("tooltip.owner.currentowner") + " "
+                            + par1ItemStack.getTagCompound().getString("ownerName"));
+        }
+    }
+
+    @Override
+    @SideOnly(Side.CLIENT)
+    public void registerIcons(IIconRegister iconRegister) {
+        this.itemIcon = iconRegister.registerIcon(CommonProxy.DOMAIN + ":frenziedFrame");
+    }
+
+    @Override
+    public ItemStack onItemRightClick(ItemStack par1ItemStack, World par2World, EntityPlayer par3EntityPlayer) {
+
+        if (EnergyItems.checkAndSetItemOwner(par1ItemStack, par3EntityPlayer)) {
+            owner = par3EntityPlayer;
+            if (par1ItemStack.getItemDamage() > 0) {
+                if (EnergyItems.syphonBatteries(par1ItemStack, par3EntityPlayer, getEnergyUsed())) {
+                    par1ItemStack.setItemDamage(par1ItemStack.getItemDamage() - 1);
+                }
+            }
+        }
+
+        return par1ItemStack;
+    }
+
+    @Override
+    public ItemStack frameUsed(IBeeHousing housing, ItemStack frame, IBee queen, int wear) {
+        // TODO Auto-generated method stub
+        if (EnergyItems.canSyphonInContainer(frame, getEnergyUsed() * wear)) {
+            if (owner != null) {
+                EnergyItems.drainPlayerNetwork(owner, getEnergyUsed() * wear);
+            }
+            // EnergyItems.syphonWhileInContainer(frame, getEnergyUsed() * wear);
```

Comment:
Remove comments

Reply:
Done, will submit as soon as I get this damn research working